doc/api/stream.md

```diff
@@ -1645,13 +1645,24 @@ parent class constructor:
 const { Writable } = require('stream');
 
 class MyWritable extends Writable {
-  constructor(options) {
-    super(options);
+  constructor({ highWaterMark, ...options }) {
+    super({
+      highWaterMark,
```

Comment:
I'd use `...options` to copy all options and just override the ones the user is not allowed to use instead of filtering them.

Reply:
I guess that's a matter of taste... what if we add a new option in the future?

Reply:
Agreed, it's not very important.